feature-libs/checkout/base/core/services/checkout-page-meta.resolver.ts

```diff
@@ -46,14 +45,8 @@ export class CheckoutPageMetaResolver
    *
    * The title from the page data is ignored for this page title.
    */
-  resolveTitle(): Observable<string> {
-    return this.cart$.pipe(
-      switchMap((c) =>
-        this.translationService.translate('pageMetaResolver.checkout.title', {
-          count: c.totalItems,
-        })
-      )
-    );
+  resolveTitle(): Observable<string | undefined> {
```

Comment:
from the first hint provided in DM, you can use resolveHeading() to change the breadcrumb title and resolveTitle will change the actual title displayed in the header and web browser